### PR TITLE
getLocation function added

### PR DESCRIPTION
**Pull request by thijsbekke**
Returns location information of the supplied location code.

### CODE REVIEW

**Comment by ameenross** [comment]
Thanks for the contribution. Please also add the low-level function to the LocationClient.

File: src/Postnl.php [hunk 1]
```diff
@@ -453,6 +453,22 @@ public function getDeliveryDate(
     }
 
     /**
+     * Returns location information of the supplied location code
+     * @param string $locationCode LocationCode information
```
Comment (ameenross) [inline src/Postnl.php]: Please leave an empty doc line before the @param lines

File: src/Postnl.php [hunk 2]
```diff
@@ -453,6 +453,22 @@ public function getDeliveryDate(
...
+     * @param string $retailNetworkID PNPNL-01 is the code that can be used for all Dutch locations
+     * @return ComplexTypes\GetLocationsResult
+     */
+    public function getLocation(
```
Comment (ameenross) [inline src/Postnl.php]: I think these params will fit on 1 line

File: src/Postnl.php [hunk 3]
```diff
@@ -453,6 +453,22 @@ public function getDeliveryDate(
...
+        $locationCode,
+        $retailNetworkID = "PNPNL-01"
```
Comment (ameenross) [inline src/Postnl.php]: Single quotes will do

**Comment by thijsbekke** [comment]
Thanks for reviewing; I processed the comments.

File: src/Postnl.php [hunk 4]
```diff
@@ -453,6 +453,21 @@ public function getDeliveryDate(
     }
 
     /**
+     * Returns location information of the supplied location code
```
Comment (ameenross) [inline src/Postnl.php]: Periods after comments and docblocks

File: src/Postnl.php [hunk 5]
```diff
@@ -453,6 +453,21 @@ public function getDeliveryDate(
...
+     *
+     * @param string $locationCode LocationCode information
+     * @param string $retailNetworkID PNPNL-01 is the code that can be used for all Dutch locations
+     * @return ComplexTypes\GetLocationsResponse
+     */
+    public function getLocation($locationCode,$retailNetworkID = 'PNPNL-01')
```
Comment (ameenross) [inline src/Postnl.php]: Space after comma

File: src/Postnl.php [hunk 6]
```diff
@@ -453,6 +453,21 @@ public function getDeliveryDate(
     }
 
     /**
+     * Returns location information of the supplied location code
+     *
+     * @param string $locationCode LocationCode information
+     * @param string $retailNetworkID PNPNL-01 is the code that can be used for all Dutch locations
```
Comment (ameenross) [inline src/Postnl.php]: While `ID` (uppercase) is used commonly (including by PostNL), it's actually an abbreviation and **not** an acronym. It should therefore be `Id`.

**Comment by thijsbekke** [comment]
What code standards are you using, so I can match them for future commits.

**Comment by danslo** [comment]
There are codesniffer grunt tasks. Don't remember them from the top of my head, but should be something like:

``` sh
npm install
grunt cs
```

And from the Gruntfile.js, it seems that the standard is PSR-2.

**Comment by ameenross** [comment]
https://github.com/DivideBV/Postnl/blob/master/CONTRIBUTING.md

**Comment by ameenross** [comment]
Do you mind testing this with the latest version of the location webservice (`2_1`)?

**Comment by thijsbekke** [comment]
I already did test it with the latest supported webservice, and it works.

**Comment by Woensdag** [comment]
Would be great if this had some more documentation or maybe even examples.

**Comment by ameenross** [comment]
Agreed, documentation could be better. However, this library does not forgo the need for PostNL's documentation itself. The responsibility of the library is relatively limited, so there wouldn't be a lot more documentation than is currently available. And much of that would be referring to PostNL's docs.

**Comment by slokhorst** [comment]
It would be nice to have a basic example for each service in the wiki, like [this one](https://github.com/DivideBV/Postnl/wiki/Multi-collo-shipment).

**Comment by thijsbekke** [comment]
@Woensdag You mean you want an example of this function or more generic of this whole library ? 

This function is not that hard, you can call it like so, 

```
$client = new Postnl(...);
$client->getLocation('175812')
```

@slokhorst be my guest ! ;)

**Comment by slokhorst** [comment]
A bit late, but here's some documentation: [wiki](/DivideBV/Postnl/wiki/Location-webservice).

**Pull request merged**